### ISSUE TITLE
Unable to use plugin via AMD

### ISSUE THREAD

**Issue by Saulzi** [Esri/esri-leaflet-renderers#135]
I am unable to use this plugin via AMD as basically the extensions to the L module are unavailable

Making the following change kind of makes the plugin work, for some reason npm test is not working on my machine ATM and I need to do some more testing before submitting a PR

```js
import L from 'leaflet';
import Esri from 'esri-leaflet';
import classBreaksRenderer from './Renderers/ClassBreaksRenderer';
import uniqueValueRenderer from './Renderers/UniqueValueRenderer';
import simpleRenderer from './Renderers/SimpleRenderer';

Esri.FeatureLayer.addInitHook(function () {
  if (this.options.ignoreRenderer) {

```

**Comment by jgravois** [comment]
thanks for reporting @Saulzi. what happens when you try and run `npm test`?

**Comment by Saulzi** [comment]
On my work laptop it gives me an error relating to code in a dependancy... I am about to try it on another machine to see if I have the same issue.

**Comment by Saulzi** [comment]
Strangely it builds and the tests run fine on my own machine... I will have a go on this machine and send a PR. may be an interesting one to write unit tests for!

**Comment by Saulzi** [comment]
The build issue relates to node v7. This will not build / run tests with the latest, it does work with the LTS of nodejs.

downgrading resolves the issue. 

```
TypeError: Cannot read property 'prototype' of undefined
    at Object.<anonymous> (/usr/local/lib/node_modules/srvlr/node_modules/socket.io/lib/store.js:35:41)
    at Module._compile (module.js:573:32)
    at Object.Module._extensions..js (module.js:582:10)
    at Module.load (module.js:490:32)
```

will probably be a case of upgrading dependency to resolve this issue..

**Comment by jgravois** [comment]
thanks for taking the time to isolate and let us know.

**Comment by Saulzi** [comment]
created #136 to track above

**Comment by Saulzi** [comment]
This was was fixed with #137
